### ISSUE TITLE
YouTube: Arrow keys don't navigate action menu immediately after manually entering Focus Mode

### ISSUE THREAD

**Issue by audiogamer22** [nvaccess/nvda#18019]
I've contacted google disability support about this and found out something interesting, I'm unable to arrow through the menu items when using NVDA as I do, even if I disabled all addons through resetting, it doesn't make a difference, but if I set NVDA to factory defaults, where NVDA automatically goes into focus mode, it works with no issues.

### Steps to reproduce:
go to youtube.com on any browser and log in, I am so if you have an account just do that to be safe, then press b until you hear, action menu button,, press space/enter, then go into focus mode by pressing NVDA plus space, you should be able to arrow down through options about the video, for example, download, save to playlist and so on, but you should find it won't work.
It will if you arrow down once after pressing space/enter on the button though.

### Actual behavior:

Nothing seams to happen.
### Expected behavior:

I should be able to arrow through the items on the screen after going into focus mode.
### NVDA logs, crash dumps and other attachments:
Added but if you need more,, let me know,, but add steps to get them if you can as I'm new to this kind of stuff.
#### NVDA installed/portable/running from source:
Installed.
#### NVDA version:
2024.4.2
#### Windows version:
24h2
#### Name and version of other software in use when reproducing the issue:
Firefox version 137.0.2 (64-bit) 137.0.2 (64-bit)
#### Does the issue still occur after restarting your computer?
Yes
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes
[nvda.log](https://github.com/user-attachments/files/19924288/nvda.log)

**Comment by audiogamer22** [comment]
@gerald-hartig Hi, does the block needs external fix thing you added mean that google need to fix it on their end?

If so; i'm writing to them about accessibility issues with youtube; would it be a good idea to show them this link?

Also, thanks for changing the title, it does sound a lot better.

**Comment by seanbudd** [comment]
Yes, Google needs to fix it. Feel free to include it in the message, thanks for reaching out to them

**Comment by audiogamer22** [comment]
@seanbudd No problem. I'll mention it and see where things go.

**Comment by audiogamer22** [comment]
So I've mentioned it and I've wrote bakc and forth with google and they said they'd get in touch when they have a solution.

It's odd thogh, I heard them use NVDA and it seams they're using an older version of youtube, the like comments are buttons as they used to be an they can type into the edit box, I asked if I could speak to the internal team but they just wrote back saying that they'd get back to me once they have a solution, you'd think they'd let  you speak to this disability internal team but no, then again what do you expect from google? It's honestly a huge shame but there's nothing that can be done.

**Comment by gerald-hartig** [comment]
@audiogamer22 I'm reopening this issue. We use the `blocked/needs-external-fix` tag for issues like these to indicate that we can't fix it on our end. This allows us to leave the issue open so that we know the issue hasn't been resolved yet and we can keep track of any external work. For example do you have a link to the issue with google that we can link here?
